content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -137,17 +137,23 @@ updateUser: (parent, args, ctx: Context) => {
 
 ## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
 
-There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats nullable values.
+There are some caveats to filtering with conditionals which might produce unexpected results. When you filter with conditionals, you might expect one result but receive another because of the way that Prisma treats nullable values.
 
 The following table provides a high-level overview of how the different operators handle 0, 1 and `n` filters.
 
-| Operator | 0 filters         | 1 filter               | n filters            |
-| -------- | ----------------- | ---------------------- | -------------------- |
-| `OR`     | return empty list | validate single filter | validate all filters |
-| `AND`    | return all items  | validate single filter | validate all filters |
-| `NOT`    | return all items  | validate single filter | validate all filters |
+| Operator | 0 filters                                  | 1 filter               | n filters            |
+| -------- | ------------------------------------------ | ---------------------- | -------------------- |
+| `OR`     | return empty list (but see the note below) | validate single filter | validate all filters |
+| `AND`    | return all items                           | validate single filter | validate all filters |
+| `NOT`    | return all items                           | validate single filter | validate all filters |
 
-This example shows how an `undefined` parameter impacts the results returned by a query that uses the [`OR`](/reference/api-reference/prisma-client-reference#or) <span class="api"></span> operator.
+<Admonition type="info">
+
+Note: If you pass an empty value, instead of `null` or `undefined`, then the `OR` operator returns all items, as it does with the `AND` and `NOT` operators.
```

Comment:
What does "empty value" mean here? Does it refer to an empty array passed as the value of `OR` property itself, or an empty object passed as the value of a specific filter, or something else?

Reply:
Good point, I'll clarify it

Reply:
Therefore, how about this change?

```suggestion
Note: If you pass an empty string, instead of `null` or `undefined`, then the `OR` operator returns all items. This is the same behaviour as with the `AND` and `NOT` operators.
```

Reply:
Ooh, I see. So, here's my understanding of what happens here: an empty string is not any kind of special value — it shouldn't be, at least — and this is not the kind of situation that you get with `null` or `undefined` where you effectively have 0 filters. Here it's a pretty normal well-formed query with one filter where you check that email contains some specific substring. It just so happens that this substring is an empty string in this case, and any string contains an empty string as a substring, so this condition is always true, and you get all records as a result. But there's nothing special about an empty string from the point of view of how filters work (it would be a bug if there is in fact), and you would probably get a very different result if you used a different condition instead of `contains`.

Reply:
Ah, I see your point. This would be expected behavior. In that case, we don't need this caveat in the docs at all.

Before I close the PR, though, there's a bit more testing I want to do to confirm or refute what the user reported. Thanks @aqrln .